vispy/gloo/util.py

```diff
@@ -101,7 +101,7 @@ def check_enum(enum, name=None, valid=None):
 
 void main(void) {
     v_uv = a_texcoord.xy;
-    gl_Position = vec4(a_position, 0, 1);
+    gl_Position = vec4(a_position, 0., 1.);
```

Comment:
crazy, how do you find these things?

Reply:
A long time ago when fixing new/random travis failures, I started a travis docker container on my local machine to debug some shader compilation failures. On the local container it either showed a better error message or I figured it out, maybe both. Some GL compilers don't like integers being mixed with floats when creating a vec.